### DISCUSSION TITLE
LG WebOS - non trovo la tipologia PLUGIN nel repository personalizzato di HA

### DISCUSSION THREAD

**Discussion by tonyx85** [indomus/forum#1106] [GENERALE - Discussioni generiche]
Ciao,
sto provando a seguire i passi dell'articolo:
[https://indomus.it/guide/integrare-e-definire-telecomandi-virtuali-tv-lg-webos-su-lovelace-ui-di-home-assistant/]

ho un problema nel passaggio di installazione dei PLUGIN, precisamente nel punto:
...aggiungere tre “repository personalizzate” di tipologia “plugin” (copia-incolla le seguenti stringhe):...

Quando vado su HACS->Integrazioni->tre puntini->Repository personalizzati
non ho l'opzione di aggiunta dei respository di tipo "plugin".
Mi manca qualche componente da installare? come posso fare? non trovo nessuna guida che lo spiega.
![NOplugin](https://user-images.githubusercontent.com/92338604/143436924-eb48fd69-8b8d-48fc-ae5a-e7a664f4ea65.png)

Grazie

**Comment by Matthewgot** [comment]
Ciao,
Ho il tuo stesso problema, sei riuscito a risolvere?

**Comment by tonyx85** [comment]
Non ancora, ho trovato altri posti di gente che ha lo stesso problema, ma
sembra che nessuno abbia la soluzione al momento

Yours sincerely / Atentamente / Cordiali Saluti / Mit freundlichen Grüßen

[image: Triband]

* Antonio Di Fonzo*


Il giorno dom 23 gen 2022 alle ore 19:49 Matthewgot <
***@***.***> ha scritto:

**Reply by Matthewgot** [reply]
Ho trovato forse una via alternativa. Se funziona te la scrivo

**Comment by tonyx85** [comment]
Alla fine ho risolto eliminando la presa da home assistant, dopo aver
riavviato HA me l'ha trovata nuovamente e l'ho riconfigurata

Yours sincerely / Atentamente / Cordiali Saluti / Mit freundlichen Grüßen

[image: Triband]

* Antonio Di Fonzo*


Il giorno mar 25 gen 2022 alle ore 09:47 Matthewgot <
***@***.***> ha scritto:

**Comment by Matthewgot** [comment]
Scusa in che senso eliminando la presa?

**Comment by tonyx85** [comment]
Sono andato in dispositivi, ho selezionato la mia presa dall'elenco dei
componenti Meross LAN, e ho fatto elimina. Poi ho riavviato Home assistant
e mi ha permesso di installarlo nuovamente senza problemi

Il ven 28 gen 2022, 21:51 Matthewgot ***@***.***> ha scritto:

**Comment by Matthewgot** [comment]
Grazie, non avevo quel problema ma mi è stato utile per trovare la soluzione

**Comment by Matthewgot** [comment]
Ne approfitto: Ho un tema scuro ed ho difficoltà a leggere i tasti. Non riesco a cambiare il colore delle scritte nei tasti. Tu sai come fare?
